instructions/FOSS_staging_list.pl

```diff
@@ -392,7 +392,7 @@
    
    {
       "dir"         => "zm-oauth-social",
-      "ant_targets" => ["publish-local"],
+      "ant_targets" => ["publish-local", "oauth-social-common-jar", "oauth-social-jar"],
       "stage_cmd"   => sub {
          SysExec("mkdir -p $CFG{BUILD_DIR}/zm-oauth-social/build/dist");
          SysExec("cp -f -rp build/zm-oauth-social-*.jar $CFG{BUILD_DIR}/zm-oauth-social/build/dist");
```

Comment:
Because of the `-*`, this will copy the `zm-oauth-social-{timestamp}.jar` created by the `publish-local` command (original fat jar) not the split jar, which you named `zm-oauth-social.jar`.

Probably remove `publish-local`, and adjust this cp source to remove the `-`.